packages/generators/generators/lib/templates/js/policy.js.hbs

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
 /**
- * `{{id}}` policy.
+ * `{{ id }}` policy.
```

Comment:
could you remove the `.` here as well to be consistent with the other files?

Reply:
Removed